content/en/docs/languages/go/instrumentation.md

```diff
@@ -876,7 +875,158 @@ meterProvider := metric.NewMeterProvider(
 
 ## Logs
 
-The logs API is currently unstable, documentation TBA.
+Logs are distinct from metrics and traces in that **there is no user-facing
+OpenTelemetry logs API**. Instead, there is tooling to bridge logs from existing
+popular log packages (e.g. slog, logrus, zap, logr) into the OpenTelemetry
+ecosystem. For rationale behind this design decision, see
+[Logging specification](/docs/specs/otel/logs/).
+
+The two typical workflows discussed below each cater to different application
+requirements.
+
+### Direct to collector (Experimental)
+
+In the direct to collector workflow, logs are emitted directly from an
+application to a collector using a network protocol (e.g. OTLP). This workflow
+is simple to set up as it doesn't require any additional log forwarding
+components, and allows an application to easily emit structured logs that
+conform to the [log data model][log data model]. However, the overhead required
+for applications to queue and export logs to a network location may not be
+suitable for all applications.
+
+To use this workflow:
+
+- Configure the OpenTelemetry [Log SDK](#logs-sdk-experimental) to export log
+  records to desired target destination (the
+  [collector][opentelemetry collector] or other).
+- Use an appropriate [Log Bridge](#log-bridge-experimental).
+
+#### Logs SDK (Experimental)
```

Comment:
```suggestion
#### Logs SDK (experimental)
```

Reply:
Same comment as above:

- Add an explicit has ID (**here and for other headings**), for example: `{#logs-sdk}`
- Move the status mention into the section prose, and then you can link to the spec definition of the term, for example:

  ```markdown
  #### Logs SDK
  
  **Status**: [Experimental](/docs/specs/otel/document-status/)
  ```

  Or only keep the "Status" in the top-level "Direct-to-Collector" section's opening paragraph?

Reply:
> only keep the "Status" in the top-level "Direct-to-Collector" section's opening paragraph?

I like this. See: https://github.com/open-telemetry/opentelemetry.io/pull/4463/commits/6f3bc5ec289df4d07aa1504e24a10f4a91324c5b

Reply:
Yes, perfect, if that what you prefer. 👍🏻